Add meeting notes for 28/1/25 (#5555)
Couchers-org/couchers: docs/meetings/dev/2025-01-28.md

# Weekly meeting

When: 28/1/25, 21:00 UTC.
Present: Aapeli, Colleen, Jesse, Nicole
Scribe: Aapeli

# Notes

* Discussion on didn't meetup feature
* Shooting the shoot

> * Discussion on map

Reviewer: ```suggestion
* Discussion on map
 - Nicole will have her design friend help out with this
```

* What everyone's working on:

>   - Aapeli: worked on cleaning up past PRs, a bunch of random things; next picking up something off the big list next, thinking about backend for volunteers and merch

Reviewer: ```suggestion
  - Aapeli: worked on cleaning up past PRs, a bunch of random things; next picking up something off the big list next, thinking about backend for volunteers and merch, got us sponsored zulip account, helped jesse with various tasks
```

  - Nicole: been doing small cleanup/deps PRs, more MUI conversions, working on tickets; working next on more MUI
  - Colleen: has been doing moral support; working on chat bubble color

>   - Jesse: did lots of support, talked with Chris, helped people get set up on Figma, volunteer recruitment; working on community builder Zulip, various UI/UX things

Reviewer: ```suggestion
  - Jesse: did lots of support, talked with Chris, Chalen, Shirley, Shane, and Jacob and helped them with various onboarding things (figma access, couchers emails, etc) + tasks they were working on, worked on volunteer recruitment - interview with several devs planned for tomorrow; working on community builder Zulip, various UI/UX things
```
